schema/function.go

```diff
@@ -251,13 +283,10 @@ func (f CallableFunctionSchema) Display() Display {
 
 func (f CallableFunctionSchema) String() string {
 	result := f.ID() + "(" + strings.Join(f.ParameterTypeNames(), ", ") + ") "
-	switch {
-	case f.DynamicTypeHandler != nil:
-		result += "dynamic"
-	case f.StaticOutputValue != nil:
-		result += string(f.StaticOutputValue.TypeID())
-	default:
-		result += "void"
+	if f.DynamicTypeHandler != nil {
+		result += "(dynamic, error)"
```

Comment:
Shouldn't the presence of `error` here be conditional on the value of `f.OutputsError`?

Reply:
As coded, it expects an error return in addition to the type returned by the handler. So it doesn't use the OutputsError in that case.

Reply:
I improved the documentation.